src/mapcat-mapview-init.js

```diff
@@ -54,8 +54,11 @@ function initView (callback, type, apiKey, layerOptions, vectorOptions, rasterOp
                     data.type = 3;
                 } else if (vectorOptions.styleSheet === 'openlayers') {
                     data.type = 2;
+                } else if (vectorOptions.styleSheet === 'navcenter') {
+                    data.type = 4;
+                    data.layers.navcenter = '';
                 } else {
-                    data.type = 1; // MapBox compatible style sheet
+                    data.type = vectorOptions.styleSheet; // custom named style sheet
```

Comment:
ez az ág volt/van valahol használva?

Reply:
egyelőre nincs, de hogy a jövőben az új style sheetekhez ezt ne kelljen bővíteni, így oldottam meg. Igazából nem is kellene semmilyen átalakítás, mert a backend sztringből is beparzolja az enumot...